### PR TITLE
Feature/pd 152 add scripts to run locally

### PR DESCRIPTION
**Pull request by jamesheward**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -45,6 +45,9 @@ aliases:
       echo 'export REACT_APP_CIRCLE_BRANCH_STR=${CIRCLE_BRANCH_STR}' >> $BASH_ENV
       echo 'export PUBLIC_URL="${REACT_APP_COGNITO_REDIRECT_URL}"' >> $BASH_ENV
       echo 'export REACT_APP_API_ENDPOINT="https://${APP_NAME}.${CLOUDFRONT_DOMAIN_NAME}/payments-poc/payments-poc-${REACT_APP_CIRCLE_BRANCH_STR}-core/ui"' >> $BASH_ENV
+      echo 'export REACT_APP_CONITO_ENDPOINT="https://${COGNITO_DOMAIN}.auth.${AWS_REGION}.amazoncognito.com"' >> $BASH_ENV
```
Comment (Steve-Conway) [inline .circleci/config.yml]: Miss-spelt Co**g**nito here. Same miss-spelling in the code that uses it, so it should work OK.

File: README.md [hunk 2]
```diff
@@ -1,18 +1,250 @@
...
+
+Dummy customer service showing how a customer woudl integrate with the service to take inbound payments
+
+## Build instructions
+
+### Frontend
+
+**Requirements**
+
+Yarn(https://yarnpkg.com/en/docs/install). We have used 1.15.2, but it should probably be fine ot use the latest stable release
```
Comment (Steve-Conway) [inline README.md]: ot -> to

File: README.md [hunk 3]
```diff
@@ -1,18 +1,250 @@
...
+
+Node JS (https://nodejs.org/en/download/). We use version 10.15.3. Latest stable shoudl work, but newer versions cause issues
```
Comment (Steve-Conway) [inline README.md]: shoudl -> should

File: README.md [hunk 4]
```diff
@@ -1,18 +1,250 @@
...
+
+Node JS (https://nodejs.org/en/download/). We use version 10.15.3. Latest stable shoudl work, but newer versions cause issues 
+
+**Instructions**
+
+After checking out the repository open the ‘frontend’ folder in the editor of your choice. Navigate to the directory in the terminal and run 
+
+`yarn install`
+
+This will run add the download all the latest version of modules into the node_modules folder.
```
Comment (Steve-Conway) [inline README.md]: This sentence lost its way somewhere.

File: README.md [hunk 5]
```diff
@@ -1,18 +1,250 @@
...
+You will need to update the .env.development file to point to the correct variables. This is used whenever the front end is 
+being server locally in development mode and tells it where to call it’s endpoints.
```
Comment (Steve-Conway) [inline README.md]: server -> served ?

File: README.md [hunk 6]
```diff
@@ -1,18 +1,250 @@
...
+
+`REACT_APP_API_ENDPOINT=http://localhost:8080` - or wherever the service you want to connect to is running
+
+`REACT_APP_COGNITO_REDIRECT_URL=http://localhost:3000`
+
+`REACT_APP_DYNAMIC_AUTH=false` - if you want to run against a local service without using cognito set this to true
+
+`REACT_APP_STATIC_SITE=true`
+
+The front end will only pick up variables is they are pre-pended with REACT_APP prefix.
```
Comment (Steve-Conway) [inline README.md]: is -> if ?

File: README.md [hunk 7]
```diff
@@ -1,18 +1,250 @@
...
+
+### Backend
+
+**Requirements**
+
+JDK 11, with the JAVA_HOME added to the PATH
+
+**Instructions**
+
+The application is build using gradle.
```
Comment (Steve-Conway) [inline README.md]: build -> built

File: README.md [hunk 8]
```diff
@@ -1,18 +1,250 @@
...
+Open a command prompt and navigate the the root of a local package (i.e. if you have followed the build instructions 
+here go to local/build/forRunningLocally).
+
+run `start-local-payments-poc.bat distributed` 
+
+This will start the core application, Kafka, an in-memory DB, the mock customer site, and the specified gateways (see configuration options). 
+Once the application is launched the default web browser will be opened with the following pages:
+- The payments poc UI
+- The interactive API page for the core application
+- A web console for teh embedded database
```
Comment (Steve-Conway) [inline README.md]: teh -> the

File: README.md [hunk 9]
```diff
@@ -1,18 +1,250 @@
...
+Open Terminal and navigate the the root of a local package (i.e. if you have followed the build instructions 
+here go to local/build/forRunningLocally).
+
+run `./start-local-payments-poc distributed`
+
+This will start the core application, Kafka, an in-memory DB, the mock customer site, and the specified gateways (see configuration options). 
+Once the application is launched the default web browser will be opened with the following pages:
+- The payments poc UI
+- The interactive API page for the core application
+- A web console for teh embedded database
```
Comment (Steve-Conway) [inline README.md]: teh -> the

File: README.md [hunk 10]
```diff
@@ -1,18 +1,250 @@
...
+This will run add the download all the latest version of modules into the node_modules folder.
+You will need to update the .env.development file to point to the correct variables. This is used whenever the front end is 
+being server locally in development mode and tells it where to call it’s endpoints.
+
+This should point to the following environment variables:
+
+`REACT_APP_COGNITO_APP_CLIENT_ID=<COGNITO_ID>`
+
+`REACT_APP_CIRCLE_BRANCH_STR=""`
+
```
Comment (sbreingan) [inline README.md]: I think we can remove this - from both the file and the readme.

Comment (sbreingan) [reply README.md]: REACT_APP_CIRCLE_BRANCH_STR

**Pull request merged**